front-end/src/App.js

```diff
@@ -2,56 +2,57 @@ import React, { useState, useEffect } from "react";
 import "./App.css";
 import SearchBar from "./components/toolbar/SearchBar";
 import { api } from "./util/api";
-import SortableRow from "./components/list/SortableRow";
-import FIXTURE from "./util/fixture.json";
+import CraftspersonRow from "./components/list/CraftspersonRow";
 import { sortByNumberOfMentees, sortByCraftspeopleWithoutMentor } from "./util/sorting";
 import { filter } from "./util/filtering";
-import Header from "./components/header/Header";
 import ButtonToolbar from 'react-bootstrap/ButtonToolbar'
 import ToggleButton from 'react-bootstrap/ToggleButton';
 import ToggleButtonGroup from 'react-bootstrap/ToggleButtonGroup';
 
 function App() {
+    const defaultSort = sortByNumberOfMentees
+    
     const [craftspeople, setCraftsPeople] = useState([]);
     const [filteredCrafspeople, setFilteredCrafspeople] = useState(craftspeople);
-    const [sortAlgorithm, setSortAlgorithm] = useState(() => sortByNumberOfMentees);
+    const [sortAlgorithm, setSortAlgorithm] = useState(() => defaultSort);
+    const [backendFetchError, setBackendFetchError] = useState(null);
```

Comment:
what do you think about naming it `craftspeopleFetchError`? I feel that `backendFetchError` is a little too wide for the context